packages/components/src/components/desktop-wrapper/desktop-wrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import { isDesktop } from '@deriv/shared';
+import React from 'react';
+
+type TDesktopProps = {
+    children: React.ReactNode;
+};
+
+const Desktop = ({ children }: TDesktopProps) => {
+    if (!isDesktop()) return null;
+    return <React.Fragment>{children}</React.Fragment>;
```

Comment:
```suggestion
    return children;
```
@jim-deriv should work just fine in the original implementation 🙂 Do you get any TS warning here?

Reply:
@maryia-binary, we had to include the `<Fragment>` here, because we were getting such errors in multiple places
<img width="1051" alt="children" src="https://user-images.githubusercontent.com/104334373/201295805-6a7e4fc2-1f91-4565-9613-4e7027e5c4b9.png">